### PR TITLE
Make planks tool materials

### PR DESCRIPTION
**Pull request by careeoki**
You can use planks to craft tools, and they each have unique stats (which will probably have to be rebalanced later)
image
Oak has the values from sticks, and is the "standard" wood type.
- Birch is faster, but with less durability
- Pine is slower, but with more durability
- Mahogany is more powerful, but would be a more dangerous biome than others (hypothetically, in the future :P)

Also updated the cobble material palette to match the newest texture.

### CODE REVIEW

File: assets/cubyz/blocks/birch_planks.zig.zon [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+		.material = .{
+		.density = 0.6,
+		.resistance = 0.8,
+		.power = 0.2,
+		.roughness = 0.7,
+		.colors = .{
+			0xff9b7a5b, 0xffbf9872, 0xffdeb287, 0xfff2c89c, 0xffffdfb7,
+		},
+	},
+	}
```
Comment (IntegratedQuantum) [inline assets/cubyz/blocks/birch_planks.zig.zon:14]: Please correctly indent this and put a comma at the end:
```zig
	.item = .{
		.material = .{
			.density = 0.6,
			.resistance = 0.8,
			.power = 0.2,
			.roughness = 0.7,
			.colors = .{
				0xff9b7a5b, 0xffbf9872, 0xffdeb287, 0xfff2c89c, 0xffffdfb7,
			},
		},
	},
```

File: assets/cubyz/blocks/birch_planks.zig.zon [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+
 	.drops = .{
 		.auto,
 	},
 	.model = "cubyz:cube",
 	.rotation = .stairs,
 	.texture = "cubyz:birch_planks",
-}
+}
```
Comment (IntegratedQuantum) [inline assets/cubyz/blocks/birch_planks.zig.zon:22]: unintended change

File: assets/cubyz/blocks/mahogany_planks.zig.zon [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+		.material = .{
+		.density = 1.0,
+		.resistance = 1.2,
+		.power = 0.3,
+		.roughness = 0.7,
+		.colors = .{
+			0xff734c2f, 0xff8e5d38, 0xffa26a3f, 0xffbd7e48, 0xffcf9459,
+		},
+	},
+	}
```
Comment (IntegratedQuantum) [inline assets/cubyz/blocks/mahogany_planks.zig.zon:14]: same as above

File: assets/cubyz/blocks/oak_planks.zig.zon [hunk 4]
```diff
@@ -1,6 +1,18 @@
...
+		.material = .{
+		.density = 1.0,
+		.resistance = 1.0,
+		.power = 0.2,
+		.roughness = 0.7,
+		.colors = .{
+			0xff835e37, 0xffa37543, 0xffbd8a4f, 0xffd3a164, 0xffe5b570,
+		},
+	},
+	}
```
Comment (IntegratedQuantum) [inline assets/cubyz/blocks/oak_planks.zig.zon:14]: same as above

File: assets/cubyz/blocks/pine_planks.zig.zon [hunk 5]
```diff
@@ -1,6 +1,17 @@
...
+		.material = .{
+		.density = 1.5,
+		.resistance = 1.2,
+		.power = 0.2,
+		.roughness = 0.7,
+		.colors = .{
+			0xff533f27, 0xff664c2e, 0xff795a37, 0xff8c6b41, 0xffa37b46,
+		},
+	},
+	}
```
Comment (IntegratedQuantum) [inline assets/cubyz/blocks/pine_planks.zig.zon:14]: same as above

**Review by IntegratedQuantum — CHANGES_REQUESTED** [review]
This is nice, thank you!
And yeah, don't worry too much about the values. They will need to be balanced in the future anyways.

There are a few formatting problems though.

**Comment by careeoki** [comment]
Okay, I fixed those issues. Let me know if it's good.

**Review by IntegratedQuantum — APPROVED** [review]
Much better, thank you.

**Pull request merged**